app/service/rest_svc.py

```diff
@@ -544,41 +547,41 @@ async def _persist_ability(self, access, ab):
         ab['tactic'] = ab.get('tactic').lower()
 
         # Validate platforms, ability will not be loaded if empty
-        if not ab.get('platforms'):
-            self.log.debug('At least one platform/executor is required to save ability.')
+        if not ab.get('platforms', dict()):
```

Comment:
The empty dictionary default isn't necessary and will be evaluated regardless of the existence of `platforms` existing in the dictionary.